examples/aggregator/gcp/pod.yaml

```diff
@@ -10,11 +10,28 @@ spec:
       image: gcr.io/oak-ci/oak-aggregator:latest
       ports:
         - containerPort: 8080
+      volumeMounts:
+        - name: tls-secret
+          mountPath: '/etc/oak-secrets'
+          readOnly: true
+      command: ['./oak_runner']
```

Comment:
FWIW I think it's fine to have the command in the Docker image (for both of them), we should still be able to override the rest of the args from here. I don't remember if it only works with ENTRYPOINT or CMD, or both, though.

Reply:
Yes, according to the [documentation](https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/#notes) it should override `entrypoint` if `command` is specified, but I don't think having it in the docker image is useful at the moment. The secret key is no longer provisioned, so the entrypoint will alwyas fail to run unless the right volume is mounted and the parameters updated to include `--private_key`.

Reply:
I think it's useful to have the binary path baked in the image, it's one less thing to have to keep in sync between the image and this file, right? Then we only deal with the flags here (we never change the actual binary that we run in an image anyways)

Reply:
I don't have a strong preference, so I added it back in.